docs/conf.py

```diff
@@ -2,57 +2,62 @@
 import os
 import sys
 
-
-sys.path.append(os.path.abspath('sphinxext'))
```

Comment:
Please don't make stylistic changes like this just because. It makes it harder for people to follow the real changes.